### PR TITLE
 Update to sku 11

### PR DESCRIPTION
**Pull request by mattcompiles**

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,8 +1,9 @@
+site/dist/playroom/
+
```
Comment (michaeltaranto) [inline .eslintignore:2]: Unnecessary given `site/dist` below.

File: .gitignore [hunk 2]
```diff
@@ -7,6 +7,7 @@ package-lock.json
 .idea/
 componentDocs.json
 lib/components/icons/*/*Svg.tsx
+site/dist/playroom/
```
Comment (michaeltaranto) [inline .gitignore:10]: As above

File: .prettierignore [hunk 3]
```diff
@@ -4,8 +4,8 @@ coverage/
 dist-storybook/
 report/
 site/dist/
-site/dist/playroom/
 # end managed by sku
 CHANGELOG.md
 .changeset/*.md
 componentUpdates.json
+site/dist/playroom/
```
Comment (michaeltaranto) [inline .prettierignore:11]: As above

File: lib/components/MenuRenderer/testHelper.tsx [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import '@testing-library/jest-dom/extend-expect';
-import React, { useState } from 'react';
+import { useState } from 'react';
+import * as React from 'react';
```
Comment (michaeltaranto) [inline lib/components/MenuRenderer/testHelper.tsx:3]: Maybe destructure `FunctionComponent` so we dont have to have this separate import

File: site/src/render.tsx [hunk 5]
```diff
@@ -1,5 +1,5 @@
 import { Render } from 'sku';
-import React from 'react';
+import * as React from 'react';
```
Comment (michaeltaranto) [inline site/src/render.tsx:2]: Can just import `ReactElement` directly here

**Review by jahredhope — APPROVED** [review]

**Pull request merged**